content/en/docs/tasks/administer-cluster/declare-network-policy.md

```diff
@@ -9,6 +9,9 @@ content_type: task
 <!-- overview -->
 This document helps you get started using the Kubernetes [NetworkPolicy API](/docs/concepts/services-networking/network-policies/) to declare network policies that govern how pods communicate with each other.
 
+{{< note >}}
+  {{% thirdparty-content %}}
```

Comment:
Is there a reason to indent the shortcode?

Reply:
Aside from legibility of nested components, no?